src/Sylius/Bundle/ApiBundle/Exception/NoFileUploadedException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+/** @experimental */
+final class NoFileUploadedException extends BadRequestHttpException
+{
+    public function __construct(
```

Comment:
```suggestion
{
    /**
     * @var array<array-key, mixed> $headers
     */
    public function __construct(
```
+ remove the entry from the baseline file.